lsr_role2collection.py

```diff
@@ -422,31 +422,31 @@ def task_cb(self, task):
         elif module_name == "include_vars":
             """
             Convert include_vars in the test playbook.
-            include_vars: path/to/linux-system-roles.ROLENAME/file_or_dir
+            include_vars: path/to/{src_owner}.ROLENAME/file_or_dir
             or
             include_vars:
-              file|dir: path/to/linux-system-roles.ROLENAME/file_or_dir
+              file|dir: path/to/{src_owner}.ROLENAME/file_or_dir
             Note: If the path is relative and not inside a role,
             it will be parsed relative to the playbook.
             To solve it, the relative path is converted to the absolute path.
             """
+            _src_owner_match = "/" + self.src_owner + "."
+            _src_owner_pattern = r".*/{0}.(\w+)/([\w\d\./]+)".format(self.src_owner)
```

Comment:
I think this should be `r".*/{0}[.](\w+)/([\w\d\./]+)".format(self.src_owner)` - that is, we want the `.` between `{0}` and `(\w+)` to match the literal dot `.` character - otherwise, it will match any character.

Reply:
Thank you, @richm. Fixed.